.travis.yml

```diff
@@ -88,6 +79,14 @@ jobs:
       local_dir: docs/_build/html
       on:
         branch: master
+  - stage: binder
+    python: '3.6'
+    before_install: skip
+    install: skip
```

Comment:
are we sure the script will get called if `install: skip` is used? We had this issue before where it might have skipped the rest of the stages...

Reply:
Good question. Luckily the answer is yes. :+1:  I tested this in a test branch and you can [see the Travis build here](https://travis-ci.org/diana-hep/pyhf/builds/476949260). You can see from the [`binder` stage](https://travis-ci.org/diana-hep/pyhf/jobs/476949265) that the call get executed properly and from [the config](https://travis-ci.org/diana-hep/pyhf/jobs/476949265/config) you can see that it used the following

```
  - stage: binder
    python: '3.6'
    before_install: skip
    install: skip
    script:
      # Use Binder build API to trigger repo2docker to build image
      - python -m webbrowser "https://mybinder.org/build/gh/diana-hep/pyhf/master"
    after_success: skip
```